src/components/Checkout/MonthlyGoodCTA.vue

```diff
@@ -12,7 +12,7 @@
 				<!-- eslint-disable-next-line max-len -->
 				Want to change lives all year long? Sign up for Monthly Good and for as little as $5 a month, you can make a lasting impact on hundreds of borrowers.
 			</p>
-			<form class="monthly-good-cta__form" @submit.prevent="null">
```

Comment:
I was getting a console warning with these, I believe `@submit.prevent` is the recommended way